tests/fixtures/on_demand_cluster_fixtures.py

```diff
@@ -104,15 +106,15 @@ def den_launched_ondemand_aws_docker_cluster(request):
     )
 
     args = {
-        "name": "aws-cpu-den",
+        "name": f"{test_rns_folder}-aws-cpu-den",
```

Comment:
what's the motivation for adding `setup_test_cluster` to the cluster name?

Reply:
Since we are using `den_tester` for running the release test, I wanted to prevent resources overlap / collision in den, in case where few release test suites are being executed in the same time.  I think it will be better that each run will create specific resources, and will not load from name (just as we do for local testing). Those resources will be cleaned-up by the den-clean up cron (runs every 24 hours). 

Potential case scenarios: 

- Two runhouse team members run release testing in the same time
- The release testing github action is executed parallelly to local release testing execution

Reply:
oh I see `test_rns_folder` is unique to the test session, I thought it was referring to the user at first